core/common/src/main/java/alluxio/wire/MasterWebUIMetrics.java

```diff
@@ -660,11 +660,11 @@ public MasterWebUIMetrics setOperationMetrics(Map<String, Metric> operationMetri
   }
 
   /**
-   * @param timeSeries the time series metrics to set
+   * @param timeSeries the time series metrics to set. The latest 20 data points will be set
    * @return the updated masterWebUIMetrics object
    */
   public MasterWebUIMetrics setTimeSeriesMetrics(List<TimeSeries> timeSeries) {
```

Comment:
If we don't need so many  point at all, we should not record so many time series into `List<TimeSeries>`, any thoughts?